pages/edit_group.py

```diff
@@ -16,6 +17,12 @@ class EditGroupPage(Base):
     _invitations_button_locator = (By.ID, 'invitations-tab')
     _invitations_tab_locator = (By.ID, 'invitations')
 
+    def __init__(self, base_url, selenium):
+        Base.__init__(self, base_url, selenium)
```

Comment:
Please replace this with:

``super(EditGroupPage, self).__init__(base_url, selenium)``